modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -71,6 +73,16 @@ def submit_power_of_attorney(poa_code)
           )
         end
 
+        def validate_2122a
+          target_veteran
```

Comment:
calling `target_veteran` to get validation on the ICN

Reply:
This stimulated a thought on checking for a file number here as our intended route is to upload the PDF to Benefits Documents, but I'm not sure how we're going to approach that, as we won't have a Claim ID. The file number would be required for that as well, hence my concern. 

I know this isn't truly necessary for the validation endpoint, nor noted in this ticket (if I can get Jira to load I'll confirm that), but if it would fail in submit, we should catch it here. I'm gathering more info before we fully implement that, so for the sake of moving items forward and getting this ready to go, this is more of a FYSA message.

Reply:
Good to know